apps/omg_watcher/test/omg_watcher/exit_processor/core_test.exs

```diff
@@ -287,9 +288,51 @@ defmodule OMG.Watcher.ExitProcessor.CoreTest do
 
       db_exits = [{utxo_pos_active, active_exit}, {utxo_pos_inactive, inactive_exit}]
 
-      expected = MapSet.new([OMG.Utxo.position(active_blknum, active_txindex, active_txoutput)])
+      expected = MapSet.new([Utxo.position(active_blknum, active_txindex, active_txoutput)])
 
       assert expected == Core.active_standard_exiting_utxos(db_exits)
     end
   end
+
+  describe "active_in_flight_exiting_inputs" do
+    test "in-flight exit info db value is formed correctly" do
```

Comment:
this is kinda weird as test though I get you'd like to verify your test functions are acting as expected.....personally I would just give up on this (or only tested locally at the beginning)

Reply:
Yeah, I agree that this test does not belongs here. I think I have a better place for the assertion. Thanks!